gdx/src/com/badlogic/gdx/math/Vector3.java

```diff
@@ -558,10 +552,7 @@ public boolean hasOppositeDirection (Vector3 vector) {
 
 	@Override
 	public Vector3 lerp (final Vector3 target, float alpha) {
-		x += alpha * (target.x - x);
-		y += alpha * (target.y - y);
-		z += alpha * (target.z - z);
-		return this;
+		return set(this.x + alpha * (target.x - x), this.y + alpha * (target.y - y), this.z + alpha * (target.z - z));
```

Comment:
This line inconsistently uses `this.x` and `x`; it can just use `x` throughout.

Reply:
I initialy wrote it without `this` but thought maybe we want to keep the original convention of using it. I'll fix it

Reply:
Alright yes I guess it doesn't worth the risk of affecting performance